src/azure-cli/azure/cli/command_modules/sqlvm/_validators.py

```diff
@@ -186,3 +190,322 @@ def validate_assessment_start_time_local(namespace):
             datetime.strptime(assessment_start_time_local, TIME_FORMAT)
         except ValueError:
             raise InvalidArgumentValueError("assessment-start-time-local input '{}' is not valid time. Valid example: 19:30".format(assessment_start_time_local))
+
+
+# pylint: disable=too-many-statements,line-too-long
+def validate_azure_ad_authentication(cmd, namespace):
+    """ Validates Azure AD authentication.
+
+        :param cli_ctx: The CLI context.
+        :type cli_ctx: AzCli.
+        :param namespace: The argparse namespace represents the arguments.
+        :type namespace: argpase.Namespace.
+    """
+
+    skip_client_validation = False
+    if hasattr(namespace, "skip_client_validation"):
+        skip_client_validation = getattr(namespace, "skip_client_validation")
+
+    if skip_client_validation is True:
+        print('Skipping client-side validation ...')
```

Comment:
If you want to warn customers, use `logger.warning()` instead of `print()`

Reply:
fixed.